applications/nutanix-ai/2.5.0/helmreleases/nai-operators.yaml

```
@@ -0,0 +1,54 @@
+---
+apiVersion: source.toolkit.fluxcd.io/v1
+kind: OCIRepository
+metadata:
+  name: nai-operators-2.5.0
+  namespace: ${workspaceNamespace}
+  labels:
+    kommander.d2iq.io/dkp-airgapped: supported
+spec:
+  interval: 1m
+  url: "${ociRegistryURL:=oci://ghcr.io}/mesosphere/charts/nai-operators"
```

```suggestion
  url: "oci://ghcr.io/mesosphere/charts/nai-operators"
```
`ociRegistryURL` is not substituted by anything.

---

addressed